### DISCUSSION TITLE
how can i get source code instead of processed HTML?

### DISCUSSION THREAD

**Discussion by razvypp** [FlareSolverr/FlareSolverr#872] [General]
Hi,

How can i get source code instead of processed HTML? for example i am scraping sitemaps and i get the processed html which is not useful.

Please let me know as it would be a great feature.

**Comment by razvypp** [comment]
anyone have any solution for this?

**Comment by Charlotte-br560** [comment]
Certainly! Obtaining source code instead of processed HTML is a valid request. For insights on implementing this feature, you might want to explore how similar functionalities are managed in projects like [Crawlbase](https://crawlbase.com/). Checking their documentation or community discussions could offer guidance on efficiently extracting source code during web scraping. Good luck with implementing this valuable addition!
